src/actions.cpp

```diff
@@ -735,6 +735,14 @@ void evm_contract::bridgeunreg(eosio::name receiver) {
     message_receivers.erase(*it);
 }
 
+
+void evm_contract::assertnonce(eosio::name receiver, uint64_t nonce) { 
+    nextnonces nextnonce_table(get_self(), get_self().value);
+
+    const nextnonce& next_nonce = nextnonce_table.get(receiver.value, "caller account has not been opened");
```

Comment:
Do not throw an error if the account has not been opened. Instead, if the entry for the account cannot be found in the `nextnonces` table, then assume the next nonce is supposed to be 0 and check that the `next_nonce` argument has that assumed value.

Reply:
done